internal/common/kafka/kafka.go

```diff
@@ -63,12 +63,21 @@ func NewConsumerEventLoop(
 	ctx context.Context,
 	consumer *kafka.Consumer,
 	schema *jsonschema.Schema,
-	handler func(context.Context, *kafka.Message),
+	handler func(context.Context, chan *kafka.Message),
```

Comment:
no need to change this interface - all the processing changes specific to the validator component should stay within the validator component

Reply:
Removed all changes from `NewConsumerEventLoop` :+1: